src/app/member/edit-member/component.ts

```diff
@@ -49,6 +49,7 @@ export class EditMemberComponent implements OnInit {
         id: this.member.id,
         name: this.member.name,
         email: this.member.email,
+        groups: this.member.groups,
```

Comment:
this field is always `undefined` irrespective you have groups or no groups in bearer token.

what is the use case to add here in EditMemberDialog?

Reply:
You're right. Let me remove it.